sdb/commands/walk.py

```diff
@@ -44,17 +52,12 @@ def call(self, objs: Iterable[drgn.Object]) -> Iterable[drgn.Object]:
             except StopIteration:
                 continue
 
-            print("The following types have walkers:")
-            print("\t%-20s %-20s" % ("WALKER", "TYPE"))
-            for type_, class_ in baked:
-                print("\t%-20s %-20s" % (class_(self.prog).names[0], type_))
-            raise TypeError("no walker found for input of type {}".format(
-                i.type_))
+            raise sdb.CommandError(
+                self.name,
+                Walk._help_message() + "no walker found for input of type {}".format(
+                    i.type_))
```

Comment:
Not a big deal but looking at your pretty-printer changes you don't place the whole help message in the CommandError string. I'm ok going other way but it would be nice to be consistent.

Reply:
In this case we need to put the entire message in the exception, because we want to be able to ignore it, in `Locator.caller()`.  If you think we should always include this in the exception, I can make a similar change in PrettyPrinter.